### PR TITLE
fix: custom cash flow mapper doesn't show any data

### PR DESCRIPTION
**Pull request by nextchamp-saqib**
- Convert `frappe.db.sql` to `frappe.qb`
- use `row.account` instead of `row.name`
   -  `row.name` was used as `account_names` to query `gl_entries`

### CODE REVIEW

File: erpnext/accounts/report/cash_flow/custom_cash_flow.py [hunk 1]
```diff
@@ -30,12 +30,12 @@ def get_mappers_from_db():
...
 	return frappe.db.sql('''
 		select cfma.name, cfm.label, cfm.is_working_capital, cfm.is_income_tax_liability,
-		cfm.is_income_tax_expense, cfm.is_finance_cost, cfm.is_finance_cost_adjustment
+		cfm.is_income_tax_expense, cfm.is_finance_cost, cfm.is_finance_cost_adjustment, cfma.account
 		from `tabCash Flow Mapping Accounts` cfma
 		join `tabCash Flow Mapping` cfm on cfma.parent=cfm.name
 		where cfma.parent in (%s)
 		order by cfm.is_working_capital
-	''', (', '.join('"%s"' % d for d in mapping_names)))
+	''', (', '.join('%s' % d for d in mapping_names)))
```
Comment (nextchamp-saqib) [inline erpnext/accounts/report/cash_flow/custom_cash_flow.py:38]: This prevents queries like `parent in (\"Temporary Opening\", )`

File: erpnext/accounts/report/cash_flow/custom_cash_flow.py [hunk 2]
```diff
@@ -124,27 +124,27 @@ def setup_mappers(mappers):
 		)
 
 		for label in account_types_labels:
-			names = [d['name'] for d in account_types if d['label'] == label[0]]
+			names = [d['account_name'] for d in account_types if d['label'] == label[0]]
```
Comment (nextchamp-saqib) [inline erpnext/accounts/report/cash_flow/custom_cash_flow.py:127]: `names` are used to query the `gl entries`, it was originally `child_row.name` which now is changed to `child_row.account`

**Comment by jigneshpshah** [comment]
@nextchamp-saqib  any idea when this will be merged to version-13 main branch  ?

**Comment by nextchamp-saqib** [comment]
It should be updated on or before 30th March

**Pull request merged**